src/routes/services.routes.js

```diff
@@ -1,11 +1,27 @@
-
-import { createNewCourse } from '../controllers/services.controllers.js';
-import { Router } from 'express';
+import { createNewFormationService, deleteFormationService, getFormationServicesByPagination,updateFormationService } from "../controllers/services.controllers.js";
+import { Router } from "express";
 import multer from "multer";
+import { validateToken } from "../helpers/jwt.js";
+
 
-const router = Router()
-const upload = multer({ dest: 'uploads/'})
+const router = Router();
+const upload = multer({ dest: "uploads/" });
 
-router.post("/create", upload.fields([{name:"courseImage"}]),createNewCourse);
+router.post(
+  "/create", validateToken,
+  upload.fields([
+    { name: "image", maxCount: 1 },
+    { name: "exhibitor_photo", maxCount: 1 },
+    { name: "supported_by_photo", maxCount: 1 },
+  ]),
+  createNewFormationService
+);
+router.get("/:skip/:taking",getFormationServicesByPagination)
```

Comment:
Suggestion: the pagination variables are often obtained from the queryParams, I think it is better to remove it from the params and use queryString params (example: /api/v1/formation-services?taking=10&skip=10

Reply:
Done